include/kalmar_math.h

```diff
@@ -568,10 +568,10 @@ namespace Kalmar
         float log2(float x) { return fast_math::log2f(x); }
 
         HCC_MATH_LIB_FN
-        float modff(float x, float *iptr) { return __ocml_modf_f32(x, iptr); }
+        float modff(float x, float *iptr) { return __ocml_modf_f32(x, (__attribute__((address_space(5))) float*)iptr); }
 
         HCC_MATH_LIB_FN
-        _Float16 modf(_Float16 x, _Float16 *iptr) { return __ocml_modf_f16(x, iptr); }
+        _Float16 modf(_Float16 x, _Float16 *iptr) { return __ocml_modf_f16(x, (__attribute__((address_space(5))) _Float16*)iptr); }
```

Comment:
Just curious, why is there a second signature `_Float16 modf(_Float16 x, _Float16* p)` in line 1316?

Reply:
.. different namespace.